software/mash/sketch/functions.nf

```diff
@@ -0,0 +1,60 @@
+/*
+ * -----------------------------------------------------
+ *  Utility functions used in nf-core DSL2 module files
```

Comment:
Need to replace this file with the one [here](https://github.com/nf-core/modules/blob/master/software/lib/functions.nf ) since it has now been updated.